portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,39 @@ function openLinkedInUrl() {
 function openGitHubUrl() { 
     window.open("https://github.com/RossJ24","_blank");
 }
+
+/**
+ * Fetches previous comments from the server.
+ */
+async function getComments(limit){
+ limit = limit.toString();
+ let res = await fetch('/data?limit='+limit); 
+ let list = await res.json();
+ let text ='';
+ for(obj of list){
+    text += obj+"\n";
+ }
+ addCommentsToDOM(text);
+}
+
+
+/**
+ * Add's the comments to the DOM
+ * @param comments The comments that were fetched from the server 
+ */
+ function addCommentsToDOM(comments){
```

Comment:
function naming convention here should be addCommentsToDom (acronym's are treated as one word, so we lower case the additional letters)